packages/babel-plugin-emotion/package.json

```diff
@@ -14,7 +14,7 @@
     "@emotion/hash": "^0.6.2",
     "@emotion/memoize": "^0.6.1",
     "@emotion/stylis": "^0.7.0",
-    "babel-core": "^6.26.3",
```

Comment:
Could you move this to `devDependencies` (with the babel 6 version) and remove all the other changes? We use babel 6 so we can test against it.